developing/languages/README.md

```diff
@@ -53,3 +55,9 @@ Important JS patterns and features to understand:
   * [How Three Dots Changed JavaScript](https://dmitripavlutin.com/how-three-dots-changed-javascript/)
 * Template Literals
   * [Template Literals](https://css-tricks.com/template-literals/)
+
+### Python versioning and management
```

Comment:
Should this be structured similar to other sections in this doc?

```md
## Python
* versioning etc [link](url)
```

Reply:
Might also be nice to throw in the other links you shared on some of the bongo stories re: django tutorials. :)

Reply:
Good idea on sectioning. Django stuff I can throw into the Web Development section.